foxtrot-core/src/main/java/com/flipkart/foxtrot/core/datastore/impl/hbase/HBaseDataStore.java

```diff
@@ -134,6 +134,7 @@ public Document save(final Table table, Document document) {
         try (org.apache.hadoop.hbase.client.Table hTable = tableWrapper.getTable(table)) {
             hTable.put(puts);
         } catch (IOException e) {
+            logger.error("Error occurred while ingesting event in HBase : {} {} {}", e, e.getCause(), e.getMessage());
```

Comment:
In case of error, {} does not work properly i think...

Reply:
reverted to log exception, removed {}